public/app/features/users/UsersTable.test.tsx

```diff
@@ -15,39 +15,45 @@ jest.mock('app/core/core', () => ({
   },
 }));
 
-const setup = (propOverrides?: object) => {
+function getProps(propOverrides?: object) {
```

Comment:
You could also use `setup` function and render the component here instead like it was done originally, so you don't need to write same code multiple times.